LICENSE-3rdparty.csv

```diff
@@ -67,5 +67,6 @@ dev,sinon,BSD-3-Clause,Copyright 2010-2017 Christian Johansen
 dev,sinon-chai,WTFPL and BSD-2-Clause,Copyright 2004 Sam Hocevar 2012–2017 Domenic Denicola
 dev,tap,ISC,Copyright 2011-2022 Isaac Z. Schlueter and Contributors
 dev,tape,MIT,Copyright James Halliday
-file,aws-lambda-nodejs-runtime-interface-client,Apache 2.0,Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
unrelated fix. This allows github to render this nicely: https://github.com/DataDog/dd-trace-js/blob/eric-navarro/validate-git-metadata/LICENSE-3rdparty.csv